app/projects/views.py

```diff
@@ -34,7 +34,7 @@
 from django.core.exceptions import ObjectDoesNotExist
 
 from hypatio.sciauthz_services import SciAuthZ
-from hypatio.scireg_services import get_user_profile
+from hypatio.scireg_services import get_user_profile, get_current_user_profile
```

Comment:
Lately I've been trying to separate imports onto individual `from ... import ...` lines so it's easier for git to track changes to them

Reply:
Good call. Will update.